content/adr/drupal-incremental-updates.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Incremental updates to Drupal"
```

Comment:
Let's think of a clearer title. When I read this at first, I had no clue what you meant or why this was an ADR.

Reply:
@Kalaiselvan88, I'm afraid it's still unclear to me if I were to just read the title. I want to convey that this is in context for preparation for major updates and not leaving it until the last minute.

Something to the effect of "Continuous handling of module updates and deprecated API".

Reply:
@hussainweb check e72861bcad573c71b3932309a8cb7e30cd6a7acb

Reply:
Accepted suggestions